kubectl-fdb/cmd/buggify_crash_loop_test.go

```diff
@@ -29,6 +29,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+var testContainerName = "test-container"
```

Comment:
What is the benefit of defining a different container name here instead of using the already defined main container and sidecar container name?